simplq/src/styles/homePage.module.scss

```diff
@@ -143,3 +143,110 @@
     font-size: small;
   }
 }
+
+.burger{
+  width: 2rem;
+  height: 2rem;
+  position: fixed;
+  top: .2rem;
+  right: 1rem;
+  z-index: 20;
+  display: none;
+  @media (max-width: 768px) {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: column nowrap;
+  }
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    background-color: #fff;
+    border-radius: .5rem;
+    transform-origin: 1px;
+    transition: all 0.3s linear;
+  
+    &:nth-child(1) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
+    &:nth-child(2) {
+      $type: open;
+      @if $type == open {
+        transform: translateX(100%);
+      } @else{
+          transform: translateX(0);
+      }
+    }
+    &:nth-child(3) {
+      $type: open;
+      @if $type == open {
+        transform: rotate(-45deg);
+      } @else{
+          transform: rotate(0);
+      }
+    }
```

Comment:
What does this do?

Reply:
this will make the hamburger button, it is not correct. I will correct this when I will work on responsiveness.

Reply:
oh right, I confused it with something else.